plugin.yml

```diff
@@ -6,6 +6,12 @@ requirements:
   - docker
 configuration:
   properties:
+    globstar:
+      type: boolean
+      default: false
+    extglob:
+      type: boolean
+      default: false
```

Comment:
Thinking about this a little bit: do we want to directly expose the implementation detail by using the `shopt` names, or would more generic keys like `extended_regex` and `recursive_regex` (or similar) be more appropriate?

Reply:
that is actually an interesting idea that would probably simplify the code a lot turning on both options if available (and adding some kind of output with a notice about it)